### ISSUE TITLE
single player lead/follow 

### ISSUE THREAD

**Issue by cryptic7s** [kolton/d2bot-with-kolbot#919]
i have multiple instances running but not sure how or if its possible to run both in same game ? 
any help?

**Comment by GabrielDias1992** [comment]
Probably you'll need to use open battle.net

**Comment by cryptic7s** [comment]
yeah, trying to run them offline on single player which is the issue. thanks for the help tho . 
if i could find a script for bots for the libs folder i could have it auto walk/level along the way. But i haven't seen any out there to do that .

**Comment by GabrielDias1992** [comment]
Did you tried the TCP/IP game ? I think this is the best solution if you want to use your single players files.

**Comment by GabrielDias1992** [comment]
What do you mean by loader ? The D2Bot executable ?

![Mode Options](https://i.imgur.com/Mbjonvz.png)

**Comment by cryptic7s** [comment]
wth ive not seen this yet i probably have an updated version . ima have to check this out.  Thanks man i really appreciate it

**Comment by cryptic7s** [comment]
bot keeps trying to run lead when trying to run map ? any advise
